spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index', type: :feature do
```

Comment:
- [ ] While you tested most of the required aspect, you probably forgot the pointed one in the image
![image](https://github.com/AleWaweru/blog-App/assets/86200356/c7f3fe70-314b-45b8-b830-605dce5839dc)

Reply:
done